frontend/src/utils/broken-link-disabler.js

```diff
@@ -0,0 +1,12 @@
+const mixin = {
+  methods: {
+    isBrokenLink(link) {
+      const linkFormat = /^(https:\/\/)(github.com|bitbucket.org|gitlab.com).*/;
```

Comment:
This logic is already performed in the backend and stored in the report `summary.json` in `supportedDomainUrlMap` (see api.js for usage). You can probably just use that here too.

Reply:
OK, I have used the data in supportedDomainUrlMap instead